docs/console-api/openapi/segments.js

```diff
@@ -723,6 +745,155 @@ module.exports = {
           }
         }
       }
+    },
+    "/segments/{segment_id}/conditional": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "segment_id",
+          "in": "path",
+          "required": true
+        }
+      ],
+      "post": {
+        "summary": "Update Segment Rules",
+        "operationId": "post-segments-segment_id-conditional",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "$ref": "../models/message.json"
+                    }
+                  }
+                },
+                "examples": {
+                  "example-1": {
+                    "value": {
+                      "message": "Rules successfully updated"
+                    }
+                  }
+                }
+              },
+              "application/xml": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "$ref": "../models/message.json"
+                    }
+                  }
+                },
+                "examples": {
+                  "example-1": {
+                    "value": "{\n    \"message\": \"Rules successfully updated\"\n}"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "integer"
+                    },
+                    "message": {
+                      "type": "string"
+                    }
+                  },
+                  "x-examples": {
+                    "example-1": {
+                      "status": 400,
+                      "message": "Segment must be of type 'id_list' to be modified"
```

Comment:
should be rule_based

Reply:
[f789c76](https://github.com/statsig-io/docs/pull/464/commits/f789c76dc3f10ac9b60d5cd3600e273944140e6a)